app/server/controllers/groupController.js

```diff
@@ -0,0 +1,233 @@
+import includes from 'lodash/includes';
+import Nexmo from 'nexmo';
+import models from '../models';
+import transporter from '../../client/utils/emailTransporter';
+
+
+export default {
```

Comment:
So according to naming conventions , this is already in the **controllers** directory so it isn't necessary to name the file **groupController.js**, you could just name it as _group.js_ so please do the same for the rest.